UnityProject/Assets/Scripts/Systems/Inventory/Pickupable.cs

```diff
@@ -205,6 +208,25 @@ private IEnumerator serverPreformInteractionLogic(HandApply interaction)
 		}
 	}
 
+	private void PickupAnim(GameObject interactor)
+	{
+		LeanTween.move(gameObject, interactor.transform, pickupAnimSpeed);
+		LeanTween.scale(gameObject, new Vector3(0, 0), pickupAnimSpeed);
+	}
+
+	[ClientRpc]
+	private void RpcPickupAnimation(GameObject interactor)
+	{
+		LeanTween.move(gameObject, interactor.transform, pickupAnimSpeed);
```

Comment:
You're still repeating the code, I intended to mean something like this:

```suggestion
		PickUpAnim( interactor);
```

Reply:
done